en/docs/tutorials/scenarios/scenario3-implementing-an-api.md

```diff
@@ -1,4 +1,63 @@
 # Scenario 3 - Implementing an API
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**RailCo has a partnership with Telecom companies to sell their train tickets as Value added services. Currently RailCo has partnerships with three such companies (Nexus, Wishque & Tenet). RailCo has a requirement to gather the stats of bookings on a daily basis from these companies. All 3 companies are exposing this data but they are exposing it in different formats. RailCo wants to develop a service which aggregates all this data and exposes it as a single API. _They first decided to create their own backend to call each of these APIs internally and write the logic to consume each service to generate the required outcome. But quickly understood the complexity of development tasks they have to undergo. For example, each of these external services uses different protocols and message formats. This would require special skills for their development teams and would increase the maintenance overhead as well.  _RailCo identified that using WSO2 API Manager and WSO2 Micro Integrator, they could do this integration in no time and expose the API with additional QoS features.**
```

Comment:
```suggestion
RailCo has a partnership with Telecom companies to sell their train tickets as value added services. Currently RailCo has partnerships with three such companies (Nexus, Wishque, and Tenet). RailCo has a requirement to gather the stats of bookings on a daily basis from these companies. All 3 companies are exposing this data but they are exposing it in different formats. RailCo wants to develop a service that aggregates all this data and exposes it as a single API. They first decided to create their own backend to call each of these APIs internally and write the logic to consume each service to generate the required outcome. However, they quickly understood the complexity of development tasks they have to undergo. For example, each of these external services use different protocols and message formats. This would require special skills for their development teams and would increase the maintenance overhead as well.  RailCo identified that using WSO2 API Manager and WSO2 Micro Integrator, they could do this integration in no time and expose the API with additional QoS features.
```